lib/Common/ChakraCoreVersion.h

```diff
@@ -54,7 +54,7 @@
 // * Does not add anything to the file description
 
 // ChakraCore RELEASE and PRERELEASE flags
-#define CHAKRA_CORE_VERSION_RELEASE 1
```

Comment:
Why did you change this? My understanding is that it is set to 1 since 1.7 is a release branch?

Reply:
You are correct, this was an oversight.